Lib/concurrent/futures/process.py

```diff
@@ -755,6 +755,11 @@ def _start_executor_manager_thread(self):
                 self._executor_manager_thread_wakeup
 
     def _adjust_process_count(self):
+        # gh-132969: avoid error if shutdown(wait=False) is called and state is reset
+        # and leaving the executor still running
```

Comment:
```suggestion
        # gh-132969: avoid leaving the executor running after
        # calling shutdown(wait=False) and state is reset.
```

I think this is what you wanted to say right?

Reply:
@picnixz this change won't avoid leaving the executor running.

I have changed the comment to the following that I think is more readable along the lines of your suggested change.

```
# gh-132969: avoid error when state is reset and executor is still running,
# which will happen when shutdown(wait=False) is called.
```